### PR TITLE
Add SymbolLoadingMetric

### PR DESCRIPTION
**Pull request by antonrohr**
Test: Unit Test
Bug: http://b/231455031

### CODE REVIEW

File: src/MetricsUploader/SymbolLoadingMetricTest.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+#include "MetricsUploader/SymbolLoadingMetric.h"
+#include "MetricsUploader/orbit_log_event.pb.h"
+
+namespace orbit_metrics_uploader {
+
+using ::testing::_;
+
+namespace {
+
+bool OrbitPerModuleSymbolLoadDataAreEqual(const OrbitPerModuleSymbolLoadData& lhs,
```
Comment (akopich) [inline src/MetricsUploader/SymbolLoadingMetricTest.cpp:22]: The error message would be uninformative, not revealing which field has mis-matched. 

I sueegest to either write a custom matcher or just replace this function with `void ExpectOrbitPerModuleSymbolLoadDataEq(lhs, rhs)` that would make three separate calls to `EXPECT_EQ`.

**Review by akopich — APPROVED** [review]
Thanks. LGTM otherwise.

**Review by karupayun — APPROVED** [review]
LGTM! :)

**Comment by antonrohr** [comment]
I am going to rebase and force push

**Comment by antonrohr** [comment]
Closing because copybara problems

**Pull request closed without merging**